### DISCUSSION TITLE
Faker Integration

### DISCUSSION THREAD

**Discussion by davidhellmann** [martinherweg/craft3-gonzo#152] [Ideas / Features] [answered]
@martinherweg 
Maybe it's a good use case to use Faker for component content. Sometimes there is no real content available and maybe it's also a good approach to test something with random data. I think it's not a big deal to integrate that.

https://fakerphp.github.io/

**Comment by davidhellmann** [comment, marked as answer]
It's integrated into CraftCMS as a dev dependency.
https://github.com/craftcms/cms/blob/develop/composer.json#L74

**Reply by martinherweg** [reply]
It's possible to use it in the ComponentConfigs so we should add examples to the documentation.
